backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -438,11 +439,18 @@ protected Map<String, Pair<String, String>> getSharedLocks() {
         if (!Guid.isNullOrEmpty(getParameters().getImageId())) {
             List<VM> vms = vmDao.getVmsListForDisk(getDiskImage().getId(), true);
             vms.forEach(vm -> locks.put(vm.getId().toString(),
-                    LockMessagesMatchUtil.makeLockingPair(LockingGroup.VM, EngineMessage.ACTION_TYPE_FAILED_VM_IS_LOCKED)));
+                    LockMessagesMatchUtil.makeLockingPair(LockingGroup.VM, getDiskIsBeingTransferredMessage())));
```

Comment:
how about checking that `getDiskImage()` doesn't return null as we do in getExclusiveLocks? (it's not required for your fix because in case it returns null we'll get NPE few lines above, but as you touch this area..)